README.md

```diff
@@ -304,28 +304,31 @@ App|Description
-[manchester_encoding](pio/manchester_encoding) | Send and receive Manchester-encoded serial.
-[onewire](pio/onewire)| A library for interfacing to 1-Wire devices, with an example for the DS18B20 temperature sensor.
+[pio_apa102](pio/apa102) | Rainbow pattern on on a string of APA102 addressable RGB LEDs.
+[pio_clocked_input](pio/clocked_input) | Shift in serial data, sampling with an external clock.
+[pio_differential_manchester](pio/differential_manchester) | Send and receive differential Manchester-encoded serial (BMC).
+[pio_hub75](pio/hub75) | Display an image on a 128x64 HUB75 RGB LED matrix.
+[pio_i2c_bus_scan](pio/i2c) | Scan an I2C bus.
+[pio_ir_loopback](pio/ir_nec) | Sending and receiving IR (infra-red) codes using the PIO.
+[pio_logic_analyser](pio/logic_analyser) | Use PIO and DMA to capture a logic trace of some GPIOs, whilst a PWM unit is driving them.
+[pio_manchester_encoding](pio/manchester_encoding) | Send and receive Manchester-encoded serial.
+[pio_onewire](pio/onewire)| A library for interfacing to 1-Wire devices, with an example for the DS18B20 temperature sensor.
 [pio_blink](pio/pio_blink) | Set up some PIO state machines to blink LEDs at different frequencies, according to delay counts pushed into their FIFOs.
-[pwm](pio/pwm) | Pulse width modulation on PIO. Use it to gradually fade the brightness of an LED.
-[spi](pio/spi) | Use PIO to erase, program and read an external SPI flash chip. A second example runs a loopback test with all four CPHA/CPOL combinations.
-[squarewave](pio/squarewave) | Drive a fast square wave onto a GPIO. This example accesses low-level PIO registers directly, instead of using the SDK functions.
-[squarewave_div_sync](pio/squarewave) | Generates a square wave on three GPIOs and synchronises the divider on all the state machines
-[st7789_lcd](pio/st7789_lcd) | Set up PIO for 62.5 Mbps serial output, and use this to display a spinning image on a ST7789 serial LCD.
+[pio_pwm](pio/pwm) | Pulse width modulation on PIO. Use it to gradually fade the brightness of an LED.
 [quadrature_encoder](pio/quadrature_encoder) | A quadrature encoder using PIO to maintain counts independent of the CPU. 
 [quadrature_encoder_substep](pio/quadrature_encoder_substep) | High resolution speed measurement using a standard quadrature encoder
-[uart_rx](pio/uart_rx) | Implement the receive component of a UART serial port. Attach it to the spare Arm UART to see it receive characters.
-[uart_tx](pio/uart_tx) | Implement the transmit component of a UART serial port, and print hello world.
-[ws2812](pio/ws2812) | Examples of driving WS2812 addressable RGB LEDs.
-[addition](pio/addition) | Add two integers together using PIO. Only around 8 billion times slower than Cortex-M0+.
+[pio_spi_flash](pio/spi) | Use PIO to erase, program and read an external SPI flash chip
+[pio_spi_loopback](pio/spi) | Use PIO to run a loopback test with all four CPHA/CPOL combinations.
```

Comment:
Not entirely clear what the naming logic is here?

| Current link text | Subdirectory | CMakeLists.txt TARGET |
| --- | --- | --- |
| pio_blink | pio/pio_blink | pio_blink |
| pio_pwm | pio/pwm | pio_pwm |
| quadrature_encoder | pio/quadrature_encoder | pio_quadrature_encoder |
| quadrature_encoder_substep | pio/quadrature_encoder_substep | pio_quadrature_encoder_substep |
| pio_spi_flash | pio/spi | pio_spi_flash |
| pio_spi_loopback | pio/spi | pio_spi_loopback |

Should the `pio/pio_blink` directory be renamed to `pio/blink` ? Should the link-text for `quadrature_encoder` be changed to `pio_quadrature_encoder` ? :shrug: